meson.build

```diff
@@ -66,6 +66,52 @@ if get_option('enable-tecio')
   subdir('externals/tecio')
 endif
 
+# PaStiX
```

Comment:
@talbring, this is the best I came up with after one day of tweaking, this version of the solver does not have a pkg-config thingy and scotch (the other dependency) needs a custom build so apt-get is not possible (I think...).
If it is not up to snuff I'll drop the PR (too much time on this already).

Reply:
Thanks of taking that effort. It is not to bother you or something like. The goal is always to make it as easy as possible for other people to use it. Don't get me wrong, but just having things integrated for the sake of it ... that's not something we want for the project in my opinion. And I know that it sometimes takes time, but it is worth it.

Having that said, it looks good to me!

Reply:
Thanks, I understand that but it is not just for the sake of integrating, structural problems do need something like this, most FEA solvers have similar solutions.

I have yet to make this work with an Intel compiler and Intel mpi, I am adding WIP to the name.